submodules/eps/EPS.py

```diff
@@ -0,0 +1,187 @@
+import logging
+import time
+import smbus
+import yaml
+
+from smbus2 import SMBusWrapper
+from submodules import telemetry
+from core.log import Log
+from core.error import Error
+# Threadhandler dependencies
+# from functools import partial
+# from core.threadhandler import ThreadHandler
+
+class EPS():
```

Comment:
Include a `has_modules` function which returns `True` if `len(self.modules) > 0`.
Before referencing any object in `self.modules` check if `has_modules` is `True` and if it is `False` raise a `RuntimeError`